src/writeData/subscriptions/components/ProgressMenuItem.tsx

```diff
@@ -0,0 +1,44 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import {Icon} from '@influxdata/clockface'
+
+interface Props {
+  active: string
+  text: string
+  type: string
+  icon: string
+  setFormActive: (any) => void
+}
+
+const ProgressMenuItem: FC<Props> = ({
```

Comment:
this should probably live in clockface some day...but I have no problem with that being a future refactoring

Reply:
so the dumplings team are working on the same component but I'd already made mine. When they make theirs generic and it's published to clockface I'll swap this out.